apps/docs/content/docs/editor.mdx

```diff
@@ -520,10 +520,32 @@ define a number of different callbacks describing how the binding should behave.
 [BindingUtil](?) reference or a [complete example of a
 binding](/examples/shapes/tools/sticker-bindings) for details.
 
-If the strucutre of your binding needs to change over time, you can provide
+If the structure of your binding needs to change over time, you can provide
 [migrations](/docs/persistence#Shape-props-migrations) describing how old stored bindings can be
 brought up to date.
 
+## Image exports
```

Comment:
side rant: we should split up the Editor page - it's sooo loooong.
but this is neither here nor there for this PR :P

Reply:
desperate to split this up 😭